include/lsst/jointcal/AstrometryFit.h

```diff
@@ -168,12 +168,40 @@ class AstrometryFit {
                                Eigen::VectorXd &grad);
 
     /**
-     * returns how many outliers were removed. No refit done. MeasOrRef can be
-     * "Meas" , "Ref", or "Meas Ref".
+     * Discards measurements and reference contributions contributing to the chi2 more than nSigmaCut,
+     * computed as:
+     * @f[
+     *    <chi2> + nSigmaCut + rms(chi2)
+     * @f]
```

Comment:
Yeah, it looks decent. I copied it from `AstrometryFit.cc` and wrapped it in the `@f[` to get rid of the doxygen errors about the `<`.